### PR TITLE
Es/update collections lesson to v1.0.0

### PR DESCRIPTION
**Pull request by edwinallenz**
Update collections lesson to v1.0.0

cc @erickgnavar

### CODE REVIEW

File: es/lessons/basics/collections.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+version: 1.0.0
 layout: page
 title: Colecciones
 category: basics
 order: 2
 lang: es
 ---
 
-Listas, tuplas, listas de palabras clave, mapas, diccionarios y combinadores funcionales
+Listas, tuplas, listas de palabras clave, mapas y combinadores funcionales.
```
Comment (eksperimental) [inline es/lessons/basics/collections.md]: I have updated this sentence in a recent commit:
https://github.com/doomspork/elixir-school/commit/3afde9ea3aeef3c114fbf7f8af58571925cc38ef#diff-ea6b1eddaf2eb6f0fad19106878c93a8R10

It should read:
> Listas, tuplas, listas de palabras clave y mapas.

File: es/lessons/basics/collections.md [hunk 2]
```diff
@@ -41,6 +41,8 @@ iex> [1, 2] ++ [3, 4, 1]
 [1, 2, 3, 4, 1]
 ```
 
+Una anotación acerca del formato (`++/2`) utilizado en Elixir (y Erlang, sobre el que está construido Elixir), el nombre de una función u operador tiene dos componentes: El nombre que le das (en este caso `++`) y su _aridad_. Aridad es una parte básica al hablar de código de Elixir (y Erlang). Es el número de argumentos que una función dada tomará (dos en este caso). Aridad y el nombre de la función son combinadas con una diagonal. Hablaremos mas acerca de esto posteriorente, este conocimiento te ayudará a entender la notación por ahora.
```
Comment (eksperimental) [inline es/lessons/basics/collections.md]: Hablaremos **más** acerca

File: es/lessons/basics/collections.md [hunk 3]
```diff
@@ -68,7 +72,7 @@ iex> tl [3.14, :pie, "Apple"]
 [:pie, "Apple"]
 ```
 
-Además de la funciones citadas, puedes usar el operador tubería `|`; veremos este patrón en futuras lecciones:
+Además de la funciones citadas, puedes usar [coincidencia de patrones](../pattern-matching/) y el operador tubería `|` para dividir una lista en cabeza y cola; veremos este patrón en futuras lecciones:
```
Comment (eksperimental) [inline es/lessons/basics/collections.md]: I think we are mixing terminology here: "operador tubería" literally translates back to English as "pipe operator", which is `|>`. We are talking about the "cons operator" `|` here. We should call it "operador cons" for now. I will investigate what's called in Spanish by other authors, and get back here later on.

Comment (eksperimental) [inline es/lessons/basics/collections.md]: It's been reported before, so after this is updated we can tick the task list here
https://github.com/doomspork/elixir-school/issues/608

I've been looking into it and there is no translation other than "operador cons"

File: es/lessons/basics/collections.md (hunk 2 above)
Comment (erickgnavar) [inline es/lessons/basics/collections.md]: ¿estos cambios no están también en 5f7e66f?, sería bueno juntar los commits para evitar futuros conflictos

File: es/lessons/basics/collections.md (hunk 3 above)
Comment (edwinallenz) [inline es/lessons/basics/collections.md]: thanks @eksperimental

**Comment by edwinallenz** [comment]
Lesson updated to v1.0.1 and already rebased to master

**Comment by edwinallenz** [comment]
I made some mistakes rebasing commits, I'll update it and let you know when is ready

File: es/lessons/basics/collections.md [hunk 4]
```diff
@@ -146,21 +159,13 @@ iex> %{foo: "bar", hello: "world"} == %{:foo => "bar", :hello => "world"}
 true
 ```
 
-## Diccionarios
-
-En Elixir, ambos, listas de palabras claves y mapas implementan el módulo `Dict`; tales se conocen colectivamente como diccionarios. Si necesitas construir tu propio almacenamiento clave-valor, implementar el módulo `Dict` es un buen lugar para empezar.
-
-El [módulo `Dict`](http://elixir-lang.org/docs/stable/elixir/#!Dict.html) provee un número de funciones útiles para interactuar y manipular esos diccionarios:
+Otra característica interesante de los mapas es que poseen su propia sintaxis para actualizar y acceder los átomos como clave.
```
Comment (eksperimental) [inline es/lessons/basics/collections.md]: It is  "__acceder a__ los átomos", but still this is not clear. 
How about: 
> Otra característica interesante de los mapas es que poseen su propia sintaxis para realizar operaciones de actualización y lectura.

I wouldn't mention "atom keys". It is a bit confusing that little bit at the end.

**Comment by doomspork** [comment]
Howdy @edwinallenz!  I wanted to follow up and see if you had time to incorporate the PR's feedback?

**Comment by edwinallenz** [comment]
Let me follow up this week

**Comment by doomspork** [comment]
Thank you @edwinallenz, you're the best!

**Comment by doomspork** [comment]
@edwinallenz could you resolve the conflict so I can merge for you?  😁

**Comment by eksperimental** [comment]
Im closing this one since it has been replaced by #1400.
Please submit a new PR if anything didnt make it through

**Pull request closed without merging**